diff --git a/wrapper/main.cpp b/wrapper/main.cpp
index c29839a..a72d101 100644
--- a/wrapper/main.cpp
+++ b/wrapper/main.cpp
@@ -252,8 +252,7 @@ constexpr struct Opt {
          valseparator(valseparator),
          valseparatorlen(valseparator ? constexprStrLen(valseparator) : 0) {}
 } opts[] = {
-  {"-mmacos-version-min", versionmin, true, false, "="},
-  {"-mmacosx-version-min", versionmin, true, false, "="},
+  {"-miphoneos-version-min", versionmin, true, false, "="},
   {"-stdlib", stdlib, true, false, "="},
   {"-arch", arch, true},
   {"-m16", arch},